docs/reference/configuration.md

```diff
@@ -262,7 +262,7 @@ logging:
   output: "-"
 
 database:
-  connection_string: "postgres://user:pass@lakefs.rds.amazonaws.com:5432/postgres"
+  connection_string: "postgres://user@lakefs-db:pass@lakefs-db.postgres.database.azure.com/postgres"
```

Comment:
I don't think this syntax is supported. https://www.postgresql.org/docs/current/libpq-connect.html
check 34.1.1.2. Connection URIs

Reply:
I have just setup an instance using this syntax :) that's why I'm suggesting this change

Reply:
Maybe there is a difference between Postgres versions?

Reply:
@nopcoder WDYT?

Reply:
Don't think we need to update the documentation - the update example uses a user name called 'user@lakefs-db' which is a specific example.